### ISSUE TITLE
incorrect root link through the flag

### ISSUE THREAD

**Issue by JOduMonT** [kalwalt/gatsby-starter-i18n-bulma#141]
Hi;
When you click on the flag and visit http://127.0.0.1:8000/it/ than click on the English flag to return to the English version it end with a double slash
http://127.0.0.1:8000/en//
note the same thing happen when you are at http://127.0.0.1:8000/en/ and click on the Italian flag.

**Comment by kalwalt** [comment]
> Hi;
> When you click on the flag and visit http://127.0.0.1:8000/it/ than click on the English flag to return to the English version it end with a double slash
> http://127.0.0.1:8000/en//
> note the same thing happen when you are at http://127.0.0.1:8000/en/ and click on the Italian flag.

Yes @JOduMonT  i saw this, but hadn't the time to fix.
I think i need to add another check in this function https://github.com/kalwalt/gatsby-starter-i18n-bulma/blob/5107c970a2798f08da5d4134ca7e63b42a1e3445/src/components/Layout.js#L51-L58

to not add the extra `/` if the link is the root `/it/` or `/en/`.

**Comment by kalwalt** [comment]
I suppose something like this (not tested at all):
```javascript
const setLangsMenu = ( langsMenu, id, basePath, jsonData) => { 
   if(id !== 'undefined'){
    if (id=='01') {
      langsMenu[0].link = `/en/';  
      langsMenu[1].link = `/it/; 
     } else {
        langsMenu[0].link = `/en/${basePath}` + getIdJsonUrl(id, 'en', jsonData) + '/';  
        langsMenu[1].link = `/it/${basePath}` + getIdJsonUrl(id, 'it', jsonData) + '/'; 
     }
  }else{  console.log("missed id in the setLangsMenu() function!");  }
};
```

**Comment by kalwalt** [comment]
@JOduMonT i'm working on this branch https://github.com/kalwalt/gatsby-starter-i18n-bulma/tree/fix-lang-switch if you want to test,  it seems that https://github.com/kalwalt/gatsby-starter-i18n-bulma/commit/be3a25b61a7722a2b86ba36c4882663225cf4993 fix the issue.

**Comment by kalwalt** [comment]
Should be solved with #145,  closing for now.

**Comment by kalwalt** [comment]
I have tested in different situations and don't completely solve the issue: the solution in the last commit works for the home page but not if you change the page from a page like `/artworks` to `/home` it will add an extra `/`. This happens clicking on the logo. I will open a new issue for this.
